packages/messenger-widget/src/components/Chat/Chat.tsx

```diff
@@ -78,7 +77,7 @@ export function Chat(props: HideFunctionProps) {
             id="chat-msgs"
             className={'chat-msgs width-fill '
                 .concat(
-                    selectedContact
+                    selectedContact && screenWidth >= 800
```

Comment:
Please make 800 a global constant so we could adjust I later without going through every file